OpenUpSA/wazimap-ng-ui
# using theme color

## Description
coloring 
* the theme icon,
* selected theme & category background 
* point markers
* clusters

according to the selected color in the theme configuration page in the admin panel

## Related Issue
https://wazimap.atlassian.net/browse/WNCM-299

## How is it tested automatically?
added automated tests into `__tests__/gui/point_data.feature` and `__tests__/utils.test.js`

## How should a reviewer test it locally
* go to admin panel
* you can use the review app https://wazimap-pr-424.herokuapp.com/admin
* in the theme configuration page change the color
* refresh the map and confirm that the theme color is applied

## Screenshots
![image](https://user-images.githubusercontent.com/53019884/191093195-a4a4582b-6513-4a12-80b9-7f9e5aa17e58.png)


## Changelog

### Added

### Updated

### Removed


## Checklist


### Pull Request


### Code Quality


### Testing
